api/src/pcapi/core/educational/factories.py

```diff
@@ -120,6 +120,16 @@ def educational_domains(
                 domains.append(domain)
             self.domains = domains
 
+    @factory.post_generation
+    def template(self, create, extracted, **kwargs):  # type: ignore [no-untyped-def] # pylint: disable=no-self-argument
```

Comment:
tu peux typer cette fonction:
```suggestion
    def template(self, create: bool, extracted: models.CollectiveOfferTemplate, **kwargs: dict):  
```

Reply:
Quelle bonne idée ! 😉